### ISSUE TITLE
British Newspaper Archive dezoomer

### ISSUE THREAD

**Issue by mostnegm** [lovasoa/dezoomify#418]
Hello lovasoa :)

I'm subscribed to BNA (access to photos requires subscription). The website offers download option but the downloadable file is less in quality than the one in the website's dezoomer.

![asasaasa](https://user-images.githubusercontent.com/61381735/87661791-8841f300-c761-11ea-9054-fde54d9df1af.jpg)

URL for the photo reference: https://www.britishnewspaperarchive.co.uk/viewer/BL/0001578/18691211/052/0011

doesn't work on dezoomify website.

(I can send you my username and pw if you need)

cheers!

**Comment by lovasoa** [comment]
Hello,
  We have a wiki page about accessing protected images. Please see: https://github.com/lovasoa/dezoomify/wiki/How-to-access-protected-pages

  You can improve the wiki page and report any hard to follow steps
